NBA-Stats-Dashboard/src/index.css

```diff
@@ -0,0 +1,53 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.App {
+  background-color: beige;
```

Comment:
Let's use CSS custom properties/variables for the colors to make it consistent.

Reply:
Fixed.